static/js/compose_validate.js

```diff
@@ -163,6 +165,40 @@ export function warn_if_mentioning_unsubscribed_user(mentioned) {
     }
 }
 
+export function clear_topic_resolved_warning() {
+    $("#compose_resolved_topic").hide();
+    $("#compose_resolved_topic").empty();
+    $("#compose-send-status").hide();
+}
+
+export function warn_if_topic_resolved() {
+    const stream_name = compose_state.stream_name();
+    const topic_name = compose_state.topic();
+
+    const sub = stream_data.get_sub(stream_name);
+
+    if (sub && topic_name.startsWith(message_edit.RESOLVED_TOPIC_PREFIX)) {
```

Comment:
@timabbott I also need to add a check here that the topic exists in stream already or not. If topic doesn't in the stream and one clicks the unresolve button, error comes. So need to see that.

This also enables us to question one thing, should we show the warning even if the topic already doesn't exist and starts with the `RESOLVED_TOPIC_PREFIX`. what are your views?